ajax/libs/pickmeup/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "pickmeup",
```

Comment:
Let's use PickMeUp which is the name of GitHub repo for git auto-update here.
Don't forget to change folder name and run `npm test` before pushing the branch.